ui/src/notebooks/context/notebook.tsx

```diff
@@ -131,6 +141,19 @@ export const NotebookProvider: FC = ({children}) => {
     [id]
   )
 
+  const scrollToPipe = useCallback(
+    (idx: number) => {
+      const targetPipe = document.getElementById(`${NOTEBOOK_PANEL_ID}${idx}`)
```

Comment:
Yeehaw vanilla JS! `ref` is probably the more reacty way to do this but also would require me changing more files

Reply:
lol we should totally change more files. this has an implicit coupling with a presentation layer that is defined somewhere else. Change one, and you have to somehow remember to change the other. Being explicit at least draws a line representing that connection in the code